src/components/Nav/Nav copy.js

```diff
@@ -0,0 +1,147 @@
+import React, { useState, useEffect } from 'react';
+import styled, { css } from 'styled-components/macro';
+import { Link, useLocation } from 'react-router-dom';
+
+function Nav() {
+  const [crrLocation, setCrrLocation] = useState();
+  const location = useLocation();
+
+  useEffect(() => {
+    setCrrLocation(location.pathname);
+  }, [location.pathname]);
+
+  useEffect(() => {
+    console.log(crrLocation);
+  }, [crrLocation]);
+
+  return (
+    <Container>
+      {location.pathname === '/' ? (
+        <LinkGroup>
+          <Link to="/">
+            <i class="fas fa-bars"></i>
+            <span>프로젝트 둘러보기</span>
+          </Link>
+          <Link to="/project-start">프로젝트 올리기</Link>
+        </LinkGroup>
+      ) : null}
```

Comment:
```suggestion
      {location.pathname === '/' && (
        <LinkGroup>
          <Link to="/">
            <i class="fas fa-bars"></i>
            <span>프로젝트 둘러보기</span>
          </Link>
          <Link to="/project-start">프로젝트 올리기</Link>
        </LinkGroup>
      )}
```
true인 경우만 표현해주면 될 것 같습니다

Reply:
네!